ghost/core/core/server/data/schema/schema.js

```diff
@@ -417,6 +417,7 @@ module.exports = {
     members: {
         id: {type: 'string', maxlength: 24, nullable: false, primary: true},
         uuid: {type: 'string', maxlength: 36, nullable: true, unique: true, validations: {isUUID: true}},
+        transient_id: {type: 'string', maxlength: 191, nullable: false, unique: true},
```

Comment:
Wonder if we can use the `uuid` instead of the `email` as the default value for `transient_id`? Then we could keep that `maxLength` to 36 instead of 191. We can then also apply validation to the field for better consistency (`isUUID: true`)